.github/workflows/cws-btfhub-sync.yml

```diff
@@ -26,25 +26,25 @@ jobs:
           docker rmi $(docker image ls -aq) >/dev/null 2>&1
 
       - name: Checkout datadog-agent repository
-        uses: actions/checkout@v4
+        uses: actions/checkout@0ad4b8fadaa221de15dcec353f45205ec38ea70b # v4.1.4
```

Comment:
I would much rather we don't use comments with the version here since it will get out of sync if we merge a dependabot PR updating the sha

Reply:
If I believe [this](https://github.com/dependabot/dependabot-core/issues/4691) I think it will be synced

Reply:
Oh that's amazing !